verifyCredentials.js

```diff
@@ -1,17 +1,17 @@
-'use strict';
+const logger = require('@elastic.io/component-logger')();
```

Comment:
why you use logger in verifyCredentials? This module support this.logger. Please, fix it